### PR TITLE
fix item dupe for eggbug, vulture, mother spider

### PR DESCRIPTION
**Pull request by youbitchoc**
hacky, should use IL hooks to disable clientside spawns

### CODE REVIEW

File: Game/RainMeadow.CreatureHooks.cs [hunk 1]
```diff
@@ -202,5 +206,36 @@ private void TentaclePlantAI_Update(On.TentaclePlantAI.orig_Update orig, Tentacl
             orig(self);
             if (!self.creature.IsLocal()) self.mostInterestingItem = mostInterestingItem;
         }
+
+        // HACK: doesn't play sounds, we should IL hook to disable just the eggs
+        private void EggBug_DropEggs(On.EggBug.orig_DropEggs orig, EggBug self)
+        {
+            if (!self.IsLocal())
```
Comment (6fears7) [inline Game/RainMeadow.CreatureHooks.cs:213]: Because we're checking OPOs won't we need a game mode check?

Comment (youbitchoc) [reply Game/RainMeadow.CreatureHooks.cs:213]: if it's a remote creature then remote will be in charge of simulating it, gamemode doesn't change this

Comment (6fears7) [reply Game/RainMeadow.CreatureHooks.cs:213]: Is it worth wrapping it in a game mode check (or at least it online mode is not null) so we don't run operations period in single player story?

Comment (youbitchoc) [reply Game/RainMeadow.CreatureHooks.cs:213]: IsLocal returns true if offline

**Pull request merged**